testsuite/MDAnalysisTests/transformations/test_rotate.py

```diff
@@ -71,17 +77,18 @@ def test_rotateby_custom_point(rotate_universes, point):
     trans = trans_u.trajectory.ts
     ref = ref_u.trajectory.ts
     vector = [1, 0, 0]
-    pos = point.reshape(3, )
+    pos = point.reshape(
+        3,
+    )
```

Comment:
```suggestion
    pos = point.reshape(3)
```
I think this is equivalent, unless I'm missing something. Unnecessary trailing commas lead to odd formatting.